recipes-workaround/storagemanager/storagemanager_git.bb

```diff
@@ -0,0 +1,89 @@
+# ============================================================================
+# RDK MANAGEMENT, LLC CONFIDENTIAL AND PROPRIETARY
```

Comment:
- Confidential header in opensourced repo?
- Licensed component in opensourced repo?  Thisnrecipe doesn't belong here.